app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -22,16 +24,27 @@ public void clickedImage(final ImageView imageView) {
         imageView.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
+                PackageManager pkManager = view.getContext().getPackageManager();
                 String url = "";
                 if (imageView.getTag().toString().equals(view.getContext().getString(R.string.linkedin))) {
                     url = speaker.getLinkedin();
-
                 } else if (imageView.getTag().toString().equals(view.getContext().getString(R.string.twitter))) {
                     url = speaker.getTwitter();
-
+                    String idname = url.substring(url.lastIndexOf('/') + 1);
+                    try {
+                        PackageInfo pkgInfo = pkManager.getPackageInfo("com.twitter.android", 0);
```

Comment:
@championswimmer OK, got it.
BTW I was digging really hard through the app and was unable find any bug/issue that hasn't been reported or assigned yet, so would it be ok if I start cleaning up the code a bit (by removing unnecessary logging statements) and fix some UI glitches that are there in the app.
Cheers.